content/installation/installing-manager.md

```diff
@@ -0,0 +1,104 @@
+1. From the CLI of your target host, run: ```sudo yum install \<rpm file link address>```
+   For example: ```sudo yum install http://cloudify-release-eu.s3.amazonaws.com/cloudify/4.3.0/release/cloudify-manager-install-4.3ga.x86_64.rpm```
+1. To change default configuration settings, edit the ```/etc/cloudify/config.yaml``` file.
+1. To install Cloudify Manager, run: ```cfy_manager install --private-ip <PRIVATE_IP> --public-ip <PUBLIC_IP>```
+
+  For more about the configure parameters, run: ```cfy_manager configure -h```
+
+## Other Installation Actions
+
+### Validating the Installation
+
+When the installation process is complete, you can use ```cfy status``` to make sure that all of the Cloudify services are running.
+Cloudify Premium customers can access the Cloudify Web Interface at: ```http://<manager_public_address>```
+
+An example output:
+{{< gsHighlight  sh >}}
+$ cfy status
+
+...
+
+Retrieving manager services status... [ip=127.0.0.1]
+
+Services:   
+   +--------------------------------+---------+
+   |            service             |  status |
+   +--------------------------------+---------+
+   | InfluxDB                       | running |
+   | Celery Management              | running |
+   | Logstash                       | running |
+   | RabbitMQ                       | running |
+   | AMQP InfluxDB                  | running |
+   | PostgreSQL                     | running |
+   | Manager Rest-Service           | running |
+   | Cloudify Stage                 | running |
+   | Webserver                      | running |
+   | Riemann                        | running |
+   | Webserver                      | running |
+   +--------------------------------+---------+
+
+   ...
+   {{< /gsHighlight >}}
+
+### Changing the Manager Parameters
+
+Before or after you install Cloudify Manager, you can change the parameters used in the installation. The configure command accepts the same CLI parameters as the install command, and it reads the config.yaml file for additional parameters.
+
+* To change installation parameters, either:
+  * Run: ```cfy_manager configure <parameters>```
+  * Edit the ```/etc/cloudify/config.yaml``` file and run: ```cfy_manager configure```
+
+The config.yaml ([View in GitHub](https://github.com/cloudify-cosmo/cloudify-manager-install)) contains more advanced configuration parameters, including:
+
+* Administrator password
+* Private and public IP address
+* External REST communication over HTTPS
+* Local path replacement for remote resources with a URL
+* Networks for Cloudify agents
+* LDAP connection information
+* SSL communication settings
+
+### Uninstalling Cloudify Manager
+
+* To uninstall the Cloudify Manager, run: ```cfy_manager remove```
+
+* To remove all of the files that the installation process extracted, run: ```yum remove cloudify-manager-install```
+
+## Next Steps
+
+After Cloudify Manager is installed, you must configure your Cloudify Manager for your environment, including:
```

Comment:
I wouldn't say "you must". I'd say "you may/can".

Reply:
Changed to can